packages/@aws-cdk/aws-servicecatalog/README.md

```diff
@@ -9,8 +9,262 @@
+```
+
+### Launch Role
+
+A launch role constraint specifies a role that the end user assumes when launching a product. You cannot have more than one of these applied to a portfolio-product association.
+
+```ts
+const portfolio = new servicecatalog.Portfolio(this, 'MyPortfolio', {
+  portfolioName: 'Portfolio Name',
+  providerName: 'Provider',
+});
+
+const product = new servicecatalog.Product(this, 'MyProduct', {
+  productName: 'Product Name',
+  owner: 'Product Owner',
+  provisioningArtifacts: [{ templateUrl: 'https://awsdocs.s3.amazonaws.com/servicecatalog/development-environment.template' }],
+});
+
+const launchRole = new iam.Role(this, 'LaunchRole', {
+  assumedBy: new iam.AccountRootPrincipal(),
+});
+
+portfolio.addLaunchRole({
+  role: launchRole,
+  product: product,
+});
+```
+
+### Provisioning Rules
+
+Provisioning rules specify certain options available to end users when launching a product, such as an `instance type`.  
+Rules are specified as a JSON string.
+
+```ts
+const portfolio = new servicecatalog.Portfolio(this, 'MyPortfolio', {
+  portfolioName: 'Portfolio Name',
+  providerName: 'Provider',
+});
+
+const product = new servicecatalog.Product(this, 'MyProduct', {
+  productName: 'Product Name',
+  owner: 'Product Owner',
+  provisioningArtifacts: [{ templateUrl: 'https://awsdocs.s3.amazonaws.com/servicecatalog/development-environment.template' }],
+});
+
+portfolio.addProvisioningRules({
+  product: product,
+  rules: {
+    'Rule1': {
+      'Assertions': [
+        {
+          'Assert': { 'Fn::Contains': [['t2.micro', 't2.small'], { 'Ref': 'InstanceType' }] },
+          'AssertDescription': 'Instance type should be t2.micro or t2.small',
+        },
+      ],
+    },
+  },
+});
+```
+
+### Stack Set Constraints
+
+A stack set constraint allows you to configure product deployment options using AWS CloudFormation StackSets. 
+You can specify multiple accounts and regions for the product launch. End users can manage those accounts and determine where products deploy and the order of deployment.
+
+
+```ts 
+const portfolio = new servicecatalog.Portfolio(this, 'MyPortfolio', {
+  portfolioName: 'Portfolio Name',
+  providerName: 'Provider',
+});
+
+const product = new servicecatalog.Product(this, 'MyProduct', {
+  productName: 'Product Name',
+  owner: 'Product Owner',
+  provisioningArtifacts: [{ templateUrl: 'https://awsdocs.s3.amazonaws.com/servicecatalog/development-environment.template' }],
+});
+
+const accounts = ['012345678901', '012345678902', '012345678903'];
+
+const regions = ['us-west-1', 'us-east-1', 'us-west-2', 'us-east-1'];
+
+const adminRole = new iam.Role(stack, 'AdminRole', {
+  assumedBy: new iam.AccountRootPrincipal(),
+});
+
+const executionRole = new iam.Role(stack, 'ExecutionRole', {
+  assumedBy: new iam.AccountRootPrincipal(),
+});
+
+portfolio.addStackSetConstraint({
+  accounts: accounts,
+  adminRole: adminRole,
+  executionRole: executionRole,
+  product: product,
+  regions: regions,
```

Comment:
Does the number of accounts must match the number of regions?

Reply:
Nope, not related at all, I will change this to avoid someone might thinking that because it might look related.